Controllers/ParkedVehiclesController.cs

```diff
@@ -0,0 +1,140 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using Garage2._0.Data;
+using Garage2._0.Models.Entities;
+using Microsoft.AspNetCore.Mvc.Rendering;
+
+namespace Garage2._0.Controllers
+{
+    public class ParkedVehiclesController : Controller
+    {
+        private readonly Garage2_0Context _context;
+
+        public ParkedVehiclesController(Garage2_0Context context)
+        {
+            _context = context;
+        }
+
+        public ActionResult Parking()
+        {
+            var vehicleTypes = Enum.GetValues(typeof(VehicleType)).Cast<VehicleType>().Select(v => new SelectListItem
+            {
+                Text = v.ToString(),
+                Value = ((int)v).ToString()
+            });
+            ViewBag.VehicleTypes = new SelectList(vehicleTypes, "Value", "Text");
+            return View();
+        }
+        [HttpPost]
+        public async Task<ActionResult> Parking(ParkedVehicle model)
+        {
+            if(ModelState.IsValid)
+            {
+                //Automatically set arrival time to current date and time
+                model.ArrivalTime = DateTime.Now;
```

Comment:
question is, if we need to set it like that if in the model we have defined the arrival time marks on the created instance.